packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "21.0.0",
+  "version": "21.1.0",
```

Comment:
Should this be a patch? `21.0.1` I think the two PRs this includes are just patches.

Reply:
Updated!